adhocracy4/maps/mixins.py

```diff
@@ -47,7 +47,7 @@ def dump_geojson(self):
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['mapitems_json'] = self.dump_geojson()
-        context['map_url'] = settings.BASE_MAP
+        context['baseurl'] = settings.MAP_BASEURL
         context['polygon'] = json.dumps(self.module.settings_instance.polygon)
```

Comment:
it is always used in the context of maps, so repeating that it is about maps should not be required. That is of course not true for the view mixins. But I am still working on converting them to template tags.